frontend/src/pages/dashboard/market-place/components/main-request/index.tsx

```diff
@@ -165,7 +165,7 @@ const MainRequest: FC<Props> = ({uid, email}) => {
 				okText="fufil request?"
 			>
 				<Typography>
-					<Title>{modalReq?.task.payerName}</Title>
+					<Title>{modalReq?.task.payerUid}</Title>
```

Comment:
Is this a placeholder? We will be replacing this with its display name, won't we?

Reply:
Oh yea yes we will